homework-submission/week3_app/src/components/FetchOrigin.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useEffect } from 'react';
+
+const Steps = () => {
+  const [loding, setLoding] = useState(true);
+  const [steps, setSteps] = useState([]);
+  const [error, setError] = useState(false);
+
+  async function getSteps() {
+    try {
+      const dataPromis = await fetch('/steps');
```

Comment:
Cool to also fetch data from your own server :-)

Reply:
thanks so much for reviewing and applying this work.